.travis.yml

```diff
@@ -7,7 +7,7 @@ go:
   - 1.6
 
 env:
-  - GO15VENDOREXPERIMENT=0
+  - GO15VENDOREXPERIMENT=1
```

Comment:
Well, we say that we support building with 1.5... I like offering people a choice, but yes, its completely arbitrary